dashboard/client/src/components/TaskTable.tsx

```diff
@@ -253,6 +255,9 @@ const TaskTable = ({
                     )}
                   </TableCell>
                   <TableCell align="center">{func_or_class_name}</TableCell>
+                  <TableCell align="center">
+                    {error_type ? error_type : "-"}
+                  </TableCell>
```

Comment:
Do we need an extra column? I already have a button to view error details.

What is "error type" anyways?

Reply:
It's shot code for task errors, e.g. WORKER_DIED, ACTOR_DIED, see some examples here (not all are related to tasks): https://github.com/ray-project/ray/blob/20a6226fe0c95e223fbab0e68ff5ef81f2f84c9c/src/ray/protobuf/common.proto#L141


But yes, we could put the error type in the Error details pop-up.